vdm-io/Joomla-Component-Builder
# Better Compiler Animations...

### Summary of Changes
- Adds the option to select between old and new gif animation in the compiler GUI area.
- Changed the compiler gif image, and improved the overall compiler GUI experience.
-  Added the feature for dynamic hashing of strings and or files.

### Testing Instructions
Test the compiler area, with both the new and old animation.

To switch between animations you must open the global **Options** and go to the compiler tab, then you will see the following option:
![image](https://user-images.githubusercontent.com/5607939/108620993-88f9e080-7438-11eb-8af0-23873d632dc9.png)

This is where you can switch between the new and old animation options.

When you select the new `3D Particle Explorations` option, you will see there is a new button in the compiler area at the top:
![image](https://user-images.githubusercontent.com/5607939/108621032-e3933c80-7438-11eb-9f8b-c37c554cb32f.png)

With this button you are able to download all the new animations to your Joomla installation in one go. This will speedup the loading of these new animations, the whole area can work, and does work without this, as it will then one by one randomly download these animations... but since there are 26 of them it can take a while before it no longer has to, so with this button the process is done once... and your done.

### Expected result
That you all like the new change we made...

### Documentation Changes Required
We will make an announcement on youtube of this improvement.
